src/Components/SmartComponents/Dashbar/Dashbar.js

```diff
@@ -1,114 +1,99 @@
 import React from 'react';
-import { Card, Grid, GridItem, Split, SplitItem, StackItem, Stack, Alert, CardBody, Text } from '@patternfly/react-core';
+import { Card, Grid, GridItem, StackItem, Stack, Alert, CardBody, Text } from '@patternfly/react-core';
 import { Main } from '@redhat-cloud-services/frontend-components/Main';
 import { SecurityIcon } from '@patternfly/react-icons';
-import { CVES_ALLOWED_PARAMS } from '../../../Helpers/constants';
+import { CVES_ALLOWED_PARAMS, impactList, CVES_DEFAULT_FILTERS } from '../../../Helpers/constants';
 import { constructFilterParameters, useUrlParams } from '../../../Helpers/MiscHelper';
-import { useDispatch } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import { changeCveListParameters } from '../../../Store/Actions/Actions';
 import { FormattedMessage } from 'react-intl';
 import messages from '../../../Messages';
+import propTypes from 'prop-types';
+import { buildActiveFilters, removeFilters } from '../../../Helpers/TableToolbarHelper';
+
+const DashbarItem = ({ title, count, impact, onLinkClick }) => {
+    return (
+        <Card isFullHeight className="card-box">
+            <CardBody>
+                <Text className="pf-u-font-size-lg pf-u-font-weight-bold">
+                    {title}
+                </Text>
+                <Text className="pf-u-font-size-lg pf-u-font-weight-bold pf-u-mt-xs">
+                    <a onClick={onLinkClick}>
+                        { Object.keys(impactList).includes(impact)
+                            ? <SecurityIcon
+                                color={impactList[impact].iconColor}
+                                size="sm"
+                                className="pf-u-mr-sm" />
+                            : null
+                        }
+                        {count}
+                    </a>
+                </Text>
+            </CardBody>
+        </Card>
+    );
+};
+
+DashbarItem.propTypes = {
+    title: propTypes.node,
+    count: propTypes.number.isRequired,
+    impact: propTypes.oneOf(impactList),
+    onLinkClick: propTypes.func.isRequired
+};
 
 const Dashbar = () => {
     const dispatch = useDispatch();
-    const [urlParameters, setUrlParams] = useUrlParams([CVES_ALLOWED_PARAMS]);
+    const [urlParameters] = useUrlParams([CVES_ALLOWED_PARAMS]);
+
+    const parameters = useSelector(
+        ({ CVEsStore }) => CVEsStore.parameters
+    );
 
     const apply = (filterParams = {}) => {
-        setUrlParams(filterParams);
         const params = constructFilterParameters(filterParams);
         dispatch(changeCveListParameters(params));
     };
 
+    const applyOnly = (filterParams = {}) => {
```

Comment:
Let's keep the name and write a comment above the function explaining what it does